site/src/main/java/uk/nhs/digital/ps/components/SeriesComponent.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.ps.components;
+
+import org.hippoecm.hst.content.beans.query.HstQuery;
+import org.hippoecm.hst.content.beans.query.HstQueryResult;
+import org.hippoecm.hst.content.beans.query.exceptions.QueryException;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+import org.hippoecm.hst.core.component.HstComponentException;
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+import org.hippoecm.hst.core.request.HstRequestContext;
+import org.onehippo.cms7.essentials.components.EssentialsContentComponent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.nhs.digital.ps.beans.Publication;
+import uk.nhs.digital.ps.beans.Series;
+
+import java.util.Iterator;
+
+public class SeriesComponent extends EssentialsContentComponent {
+
+    private static final Logger log = LoggerFactory.getLogger(SeriesComponent.class);
+
+    @Override
+    public void doBeforeRender(final HstRequest request, final HstResponse response) throws HstComponentException {
+        super.doBeforeRender(request, response);
+
+        HstRequestContext requestContext = request.getRequestContext();
+        HippoBean contentBean =  requestContext.getContentBean();
+
+        if (contentBean.isHippoFolderBean()) {
```

Comment:
else log a warning?

Reply:
Done. An error is logged and the user is notified - it's unlikely that they would ever see this particular  message (depending on site mapping such misconfiguration may actually result in a default/home page being displayed instead) but, in case it slips through, they will have clear indication that there is a problem so that they can flag it to admins.

Admittedly, the error reporting mechanism is a bit crude at the moment but we don't have nothing better until we establish a common pattern or strategy for reporting errors within the Site.